frontend/src/metabase/ui/components/buttons/Button/Button.styled.tsx

```diff
@@ -134,7 +134,7 @@ export const getButtonOverrides = (): MantineThemeOverride["components"] => ({
               color: hoverColor,
               backgroundColor: "transparent",
             },
-            "&:disabled": {
+            "&:disabled, &[data-disabled=true]": {
```

Comment:
To use the button's disabled theming with `data-disabled` we need to add this here

Reply:
It's a quirk of Mantine when using a tooltip with a disabled button: https://v6.mantine.dev/core/button/#disabled-button-with-tooltip

Ofc an `ActionIcon` is also a button so it applies here too